src/components/ae-overscreen/ae-overscreen.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="ae-overscreen" :class="[{ shadow }, align]">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ae-overscreen',
+  props: {
+    /**
+     * Applies a linear shadow gradient to the overscreen
+     */
+    shadow: Boolean,
+
+    /**
+     * Align the elements either left or right, default alignment is center.
+     * Available values are: `left, right`
+     */
+    align: {
+      type: String,
+      validator: value => [
+        'left',
+        'center',
+        'right',
+      ].includes(value),
+      default: 'center',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/globals';
+
+.ae-overscreen {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  position: absolute;
```

Comment:
As I understand, this component should be at the bottom of screen, so we should use `fixed` position. You can override it back to `absolute` in examples. Or we can drop position at all, like in AeBanner component.

Reply:
This is a top-level component, and yep, the fixed position makes more sense now.